litecommands-annotations/test/dev/rollczi/litecommands/annotations/async/AsyncCommandTest.java

```diff
@@ -24,11 +23,19 @@
 
 class AsyncCommandTest extends LiteTestSpec {
 
+    private static final String mainThreadName = "main_thread";
+    private static final String asyncThreadName = "async_thread";
+
     private static final int DELAY = 500;
     private static final int MARGIN = 200;
 
     static LiteTestConfig config = builder -> builder
-        .scheduler(new SchedulerExecutorPoolImpl("test", 1))
+        .scheduler(
```

Comment:
Please revert it, this unit test is used to check a case with the specific scheduler.